### PR TITLE
webpack setup, server, schema, and inserting mock data into database

### PR DESCRIPTION
**Pull request by rtsuzaki**
https://trello.com/c/Mbiz0cXa/7-s-add-webpack-setup

https://trello.com/c/0l7ChgrW/2-s-create-schemasql

https://trello.com/c/kUf8E7l3/8-s-add-data-to-database

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "parser": "babel-eslint",
+  "plugins": [
+    "react"
+  ],
+  "rules": {
+    "max-len": [1, 120, 2, {"ignoreComments": true}],
+    "react/prop-types": [2]
+  },
+  "extends": ["airbnb-base"]
```
Comment (clfhhc) [inline .eslintrc]: use "airbnb" if you are using react

File: client/src/components/App.jsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  }
+
+  render() {
+    return (
+      <div>RENDERED APP</div>
+    )
+  }
+}
+
+export default App;
```
Comment (clfhhc) [inline client/src/components/App.jsx]: new line

File: database/index.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+connection.connect((err) => {
+  if(err) {
+    console.log('Error connecting to the database-->', err)
+    return;
+  }
+  console.log('Connection to database established')
+})
+
+const insertRecords = (sql, data) => {
```
Comment (clfhhc) [inline database/index.js]: the function definition suggests it does all sorts of things. use better naming

File: mockData/generateMockData.js [hunk 4]
```diff
@@ -71,4 +73,23 @@ let allPrices = getNumberForAllEntries(50, 750, 600);
...
 let allNumberOfReviews = getNumberForAllEntries(0, 500, 600);
-let allUrls = getRoomPicUrl(600);
\ No newline at end of file
+let allUrls = getRoomPicUrl(600);
+
+let columnData = [allRoomNames, allPrices, allNumberOfRooms, allRatings, allNumberOfReviews, allUrls]
+
+let createRecords = (columns) => {
+  let records = [];
+  for (var i = 0; i < columns[0].length; i++) {
```
Comment (clfhhc) [inline mockData/generateMockData.js]: airbnb style: use higher oder functions, first columns[0].map, then columns.map

File: mockData/generateMockData.js [hunk 5]
```diff
@@ -53,7 +55,7 @@ loremIpsum = [
 
 const getNameForAllEntries = (words, numberOfEntries) => {
   let allWords = [];
```
Comment (LoganAHart) [inline mockData/generateMockData.js]: I think the style guidelines suggest we use const for reference types, and const unless we are changing a reference/have to change a reference - https://github.com/airbnb/javascript#references

**Pull request merged**